keras_cv/layers/preprocessing/random_rotation_test.py

```diff
@@ -99,3 +99,43 @@ def test_ragged_bounding_boxes(self):
             ragged_rank=1,
         )
         self.assertAllClose(expected_output, output["bounding_boxes"])
+
+    def test_augment_sparse_segmentation_mask(self):
+        classes = 8
+
+        input_images = np.random.random((2, 20, 20, 3)).astype(np.float32)
+        # Masks are all 0s or 8s, to verify that when we rotate we don't do bad
+        # mask interpolation to either a 0 or a 7
+        masks = np.random.randint(2, size=(2, 20, 20, 1)) * (classes - 1)
+        inputs = {"images": input_images, "segmentation_masks": masks}
+
+        # Attempting to rotate a sparse mask without specifying classes fails.
+        bad_layer = RandomRotation(factor=(0.25, 0.25))
+        with self.assertRaises(ValueError):
```

Comment:
Let's also assert the error message here.

Reply:
Done -- thank you for the review!